src/components/sheet/SlackSheet.tsx

```diff
@@ -45,7 +45,8 @@ const Container = styled(Centered).attrs({
               ? deviceHeight - contentHeight
               : 0,
       }),
-  ...(IS_ANDROID ? { borderTopLeftRadius: 30, borderTopRightRadius: 30 } : {}),
+  borderTopLeftRadius: 30,
+  borderTopRightRadius: 30,
   backgroundColor: backgroundColor,
```

Comment:
can I do this? is a square on ios if not, but only when the height is not 100% idk why